### PR TITLE
Fix profiler_kineto Clang errors

### PR DESCRIPTION
**Pull request by sraikund16**
Summary: There are clang errors in profiler_kineto. It would probably be a good idea to fix them as the file is already quite dense.

Test Plan: Make sure all on Phabricator all tests under static_tests/lint_root pass

Differential Revision: D58431005
@diff-train-skip-merge

### CODE REVIEW

File: torch/csrc/autograd/profiler_kineto.cpp [hunk 1]
```diff
@@ -47,8 +47,7 @@ __attribute__((weak)) int acc_get_device_type() {
 #endif // USE_KINETO
 
 namespace torch {
-namespace autograd {
-namespace profiler {
+namespace autograd::profiler {
```
Comment (aaronenyeshi) [inline torch/csrc/autograd/profiler_kineto.cpp:50]: do we need to combine these? I feel the normal convention in pytorch separates each namespace into their own lines

**Review by aaronenyeshi — CHANGES_REQUESTED** [review]
Please check lintrunner error

File: torch/csrc/autograd/profiler_kineto.cpp (hunk 1 above)
Comment (sraikund16) [reply torch/csrc/autograd/profiler_kineto.cpp:50]: yes it fails lint_root with modernize-concat-nested-namespaces

Comment (sraikund16) [reply torch/csrc/autograd/profiler_kineto.cpp:50]: We can do whichever you think is more important but I'd rather get rid of all the static errors whenever a diff touches this file

**Review by aaronenyeshi — APPROVED** [review]
LGTM!

**Pull request closed without merging**